pkg/k8s/synced/crd.go

```diff
@@ -42,6 +42,9 @@ func agentCRDResourceNames() []string {
 		CRDResourceName(v2.CCNPName),
 		CRDResourceName(v2.CNName),
 		CRDResourceName(v2.CIDName),
+		CRDResourceName(v2.CIDName),
+		CRDResourceName(v2alpha1.BGPPName),
+		CRDResourceName(v2alpha1.BGPPoolName),
```

Comment:
While reviewing some of this code in the tree, I noticed a couple of problems with this:

* CIDName should not be duplicated.
* The BGP resources should not be unconditionally registered like this, they should be under their own dedicated conditional like the examples below.